src/OrbitGl/TimeGraph.cpp

```diff
@@ -365,6 +365,22 @@ void TimeGraph::ProcessPageFaultsTrackingTimer(const TimerInfo& timer_info) {
   track->OnTimer(timer_info);
 }
 
+orbit_gl::CaptureViewElement::EventResult TimeGraph::OnMouseWheel(
+    const Vec2& mouse_pos, int delta, const orbit_gl::ModifierKeys& modifiers) {
+  if (delta == 0) return EventResult::kIgnored;
+
+  const float delta_normalized = delta < 0 ? -1.f : 1.f;
+
+  if (modifiers.ctrl) {
+    VerticalZoom(-delta_normalized, mouse_pos[1]);
+  } else {
+    double mouse_ratio = mouse_pos[0] / GetWidth();
+    ZoomTime(-delta_normalized, mouse_ratio);
```

Comment:
I think this is a better place to invert it, because the caller simply passes down the delta that comes from the QT MouseWheel event - it's the `TimeGraph` that decides on how to use it. But maybe we should invert the meaning of delta in `ZoomTime` and `VerticalZoom` instead. I wanted to keep the changes to the logic small and just move the reponsibility, but maybe it's worth fixing this in this PR.

Reply:
Yes, either is fine. I only found strange the inconsistency